editor/svgcanvas.js

```diff
@@ -4442,7 +4442,7 @@ var convertToGroup = this.convertToGroup = function(elem) {
 //
 // Returns:
 // This function returns false if the set was unsuccessful, true otherwise.
```

Comment:
Please update the docs of this method to mention the optional preventUndo parameter

Reply:
I've added the documentation for it now.